libbeat/processors/dns/dns_test.go

```diff
@@ -151,10 +151,40 @@ func TestDNSProcessorTagOnFailure(t *testing.T) {
 
 	v, _ := event.GetValue("tags")
 	if assert.Len(t, v, 1) {
-		assert.ElementsMatch(t, v, p.config.TagOnFailure)
+		assert.ElementsMatch(t, v, p.TagOnFailure)
 	}
 }
 
+func TestDNSProcessorDisabledCache(t *testing.T) {
```

Comment:
I don't think this test case adds any additional coverage, or at least its not validating that caching is disabled. I think you have already covered everything with `TestDisabledCache` so this case can be removed.

Reply:
reverted in https://github.com/elastic/beats/pull/44932/commits/5bfe69269c4f2e6a30aadbd5517c3059d500242f